packages/trader/build/constants.js

```diff
@@ -50,19 +49,7 @@ const rules = (is_test_env = false, is_mocha_only = false) => [
     ...(is_test_env && !is_mocha_only
         ? [
               {
-                  test: /\.(js|jsx)$/,
-                  exclude: /node_modules|__tests__|(build\/.*\.js$)|(_common\/lib)/,
-                  include: /src/,
-                  loader: 'eslint-loader',
-                  enforce: 'pre',
-                  options: {
-                      formatter: require('eslint-formatter-pretty'),
```

Comment:
did you move or remove all of this configuration from line 53 to 65?
don't wee need them?